src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -474,6 +474,18 @@ struct mlx5_cqe64 *uct_ib_mlx5_iface_cqe_unzip(uct_ib_mlx5_cq_t *cq)
     return title_cqe;
 }
 
+static void
+uct_ib_mlx5_update_cqe_compression_stats(uct_ib_iface_t *iface, uct_ib_mlx5_cq_t *cq)
```

Comment:
maybe do it inside uct_ib_mlx5_iface_cqe_unzip_fill_unique?
it already checks req/resp case

Reply:
It will cause CQE zipping calculation inside `uct_rc_mlx5_iface_commom_clean` function. If we assuming that the main use case of these counters is calculating ratio of compressed CQEs among all received CQEs, then taking zipped CQE from `uct_rc_mlx5_iface_commom_clean` in attention will break the ratio since we do not count usual (non-compressed) CQEs handled there.

I can implement the logic of incrementation only if passed iface argument isn't NULL, but it will make the code less readable. And maybe we want to calculated zipped CQE inside `uct_rc_mlx5_iface_commom_clean` as a separate counter, WDYT?

Also, to increment these counters inside `uct_ib_mlx5_iface_cqe_unzip_fill_unique` we need to pass iface to this function (and to `uct_ib_mlx5_iface_cqe_unzip` too) just to increment statistic counters. I checked that in the current implementation, compiler likely will erase the `uct_ib_mlx5_update_cqe_compression_stats` function if statistic is disabled ([godbold example](https://godbolt.org/z/cWhKGxv1T)). So does this change worth passing additional variable only for it and possible increasing of arguments footprint size?